recipes/ignition-tools/all/conanfile.py

```diff
@@ -88,18 +99,59 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "share"))
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.recipe_folder, "..", "source"))
```

Comment:
🤔 This looks out of place

I am not familair with this project so can you please share where this comes from?

Reply:
Hi @prince-chrismc . thanks for your questions and feedback. This library is a dependency of [gazebo classic](http://classic.gazebosim.org/) which is developed by open source robotics foundation (OSRF) and is used for robot simulation . Here's the github link to [ignition-tools](https://github.com/gazebosim/gz-tools)

Reply:
sorry for the late reply. this might have been copy paste from my local files in another recipe. I removed it